---
status:
    Rejected
description:
    Experiment not yet completed
---


# 2021-02-02 RNA Isolation and Sequencing of WT

## Purpose
The purpose of this experiment was to validate transcriptomic measurements of 
the wild-type NCM3722 grown in glucose minimal media. 

## Materials


## Materials

### Growth Media
| **Label** | **Buffer Base** | **Carbon Source & Concentration** |
|:--:|:--:|:--:|
| acetate | N-C- + micronutrients | 30 mM acetate |

### Strains 
| **Label** | **Parent Strain**|  **Genotype** | **Location(s)**|
|:--: | :--:| :--:| :--:|
| wt | NCM3722 | wt *E. coli* strain NCM3722 | `GC001`|

## Notes 

## Results

## Protocol

### Culturing procedure
1. A 3 mL culture of N-C- minimal medium supplemented with glucose was inoculated 
with a single colony from a fresh (< 2 week old) plate. A small amount of LB (≈ 3 µL)
was also added to reduce the lag time. 

2. The culture was allowed to grow for ≈ 5 hours at 37 °C with shaking at 250 rpm
until the OD_600nm_ was between 0.3 and 0.4. 

3. The culture was diluted 1:10 fold into prewarmed media and returned to the
waterbath shaker. Optical density measurements were taken every 15 to 30 minutes 
until the culture reached an OD_600nm_ of **XXX**.

4. At this point, 2 mL of culture was harvested for RNA isolation

### RNA Extraction
1. The 2 mL of extracted culture was pelleted for 30 seconds at 16000 x g.
2. The supernatant was rapidly removed and the pellet was suspended in 500 µL of 
a 50:50 water-Trizol mix by vortexing for 30 - 60 seconds. The solution was 
allowed to rest at room temperature for 5 min.
3. An aliquot of 25 µL of chloroform was then added to the Trizol mix and vigorously 
vortexed for 30 seconds. The mixture was then allowed to rest at room temperature 
for 5 min with occasional manual inversion. From this step onward, only RNAase-free 
equipment (such as tips, tubes, water) was used
4. The mixuture was inverted one more time and then spun at 4 °C for 1 minute at 
16000xg.
5. The aqueous-phase (supernatant) was carefully extracted and transferred to
a clean RNA-ase free 1.5 mL Eppendorf tube. This was about 500 µL.
6. An aliquot of 500 µL 200 proof ethanol was added to the isolated aqueous phase 
and mixed via pipetting. 
7. The entire volume (≈ 1 mL) was transferred to a Zymo RNA Clean and Concentrate 
filter column.
8. The column was spun at 16000xg for 1 min at 4° C. The flow-through was discarded.
9. 400 µL of RNA Prep buffer was added to the column and spun for 1 min at 16000 xg 
at 4° C. The flow-through was discarded.
10. 700 µL of RNA Wash buffer (with ethanol added) was added to the column and was 
spun at 16000xg for 1 min at 4 °C. The flow-through was discarded.
11. 400 µL of RNA Wash buffer (with etahnol added) was added to the column and 
was spun at 16000xg for 1 min at 4 °C. The flow-through was discarded.
12. 15 µL of RNAase-free water was added to the column and was transferred to an 
RNAase-free 1.5 mL eppendorf tube. This was then spun for 1 min at 
16000xg at 4° C and the column was then discarded. 
13. 5 µL of the elutant was immediately removed and transferred into an RNAase-free
1.5 mL eppendorf tube for concentration determination. The remainder of the elutant
was immediately placed on dry ice and was later moved to the -80 °C freezer.
14. 1 µL of the transferred elutant was added to the Nanodrop to quantify the 
concentration. The isolated RNA was at a concentration of **XX** with 
a 280/260 absorbance ratio of **XX**.
15. The remainder of the sample was submitted to the Stanford PAN facility for 
quality control quantification. ß